src/drivers/serial/am85c30/am85c30.c

```diff
@@ -112,13 +110,12 @@ static int am85c30_getc(struct uart *dev) {
 	while (am85c30_has_symbol(NULL) == 0) {
 	}
 
-	r1 = am85c30_com_inb_command(port, AM85C30_RR1);
-	ch = am85c30_com_inb(port+1);
+	r1 = am85c30_com_inb_command((void*)(uintptr_t)port, AM85C30_RR1);
```

Comment:
Probably, declare port as `uintptr_t ` in that case? By the way, you already use just `(void*) port` in am85c30_has_symbol